README.md

```diff
@@ -9,7 +9,7 @@ output of [coverage.py](https://coverage.readthedocs.io). Example usage is shown
 ## Installing
 The simplest way to install is to use pip:
 ```
-pip install -i https://test.pypi.org/simple/ coveragecheck
+pip install coveragecheck==0.0.2
```

Comment:
We want to drop the version number. When you use pip, it will install head version by default, and that's what we want users to use